### DISCUSSION TITLE
Extrapolation of Papp to specific intestinal permeability - how to activate the customized calibration method in PK-Sim?

### DISCUSSION THREAD

**Discussion by irina-antonescu** [Open-Systems-Pharmacology/Forum#862] [Questions & Problems] [answered]
Hi OSP community,

Can anyone help me with advice on how to input experimental apparent permeability (Papp) values for investigated compound + reference compound in PK-Sim? 

Manual states:  (https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#adme-properties  --> ADME properties --> absorption --> Calculation of Specific Intestinal Permeabilities)

"In addition to the calculated specific intestinal permeability, experimentally determined permeabilities, e.g. from Caco-2-cell permeability assays can be used. However, due to the large inter-laboratory variability in Caco-2 permeations, a proper calibration of the measured in vitro values and the calculated in silico permeabilities for a defined set of compounds is necessary. If experimentally determined values for intestinal permeabilities are available and the customized calibration method has been implemented in PK-Sim®, this option is then available in the drop-down menu in the Calculation methods window."

I am unable to find the " drop-down menu in the Calculation methods window." specified in the instructions above.

Answers to any of these questions would additionally be helpful:

Which reference compounds has the Papp calibration performed for in PK-Sim? (low - medium - high permeability)
Which cellular models have the reference PK-Sim Papp values been measured in? 
What is the equation that derives intestinal permeability from Papp? 

Thanks!

**Comment by StephanSchaller** [comment]
Dear @irina-antonescu ,

Hope this screenshot will help:
![image](https://user-images.githubusercontent.com/25477643/104584940-a1215780-5663-11eb-9e7a-afed2091bd1a.png)
- you can access/see alternative calculated values using the drop-down (marked with green)
  - a mouse-over on the publication names will give you more details
- you can then add your additional entries using the "+" (marked in yellow)

Required Infos on your additional questions can be found in the listed publications.

Hope this helps,
Stephan

**Comment by irina-antonescu** [comment]
Dear Stephan,

Unfortunately those options do not appear when I click the drop down menu.
Here is a screenshot of what I see:

![image](https://user-images.githubusercontent.com/77011727/104585962-1c373d80-5665-11eb-89de-1bd9f519b31d.png)

Is it perhaps a different version that supports this tool? I am using PK-Sim version 9 update 1.

Thanks,
Irina

**Comment by AndreDlm** [comment]
Dear @irina-antonescu ,

the different values shown in @StephanSchaller's plot are calculated from different lipophilicity values. If you add another value for the drug's lipophilicty, a second calculated value for the specific intestinal permeability will be displayed.

Currently, there's no option in PK-Sim to input experimental apparent permeability (Papp) values and convert that into a specific intestinal permeability. I remember this issue has been discussed some time ago [here](https://github.com/Open-Systems-Pharmacology/Forum/issues/23#issuecomment-308669758), maybe that helps.
In any case, the manual should be adjusted to avoid misunderstandings.

André

**Comment by irina-antonescu** [comment]
Dear @AndreDlm,

Thanks for the clarification. I agree it would be good to remove that.

Irina

**Comment by Yuri05** [comment, marked as answer]
In fact PK-Sim has built-in calibration for CACO/PAMPA assays in the form
`Intestinal permeability (transcellular) = 10 ^ (m * log10 (Specific intestinal permeability (transcellular)) + b)`
(with **assay-specific calibration parameters** `m` and `b`)

The only problem is that the calibration parameters (m and b) are currently **hidden** in PK-Sim and thus cannot be changed by user. I have just created an issue in PK-Sim for this (https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1631)

As **per default** calibration parameters are defined as `m=1` and `b=0`  which corresponds to "no calibration":
`Intestinal permeability (transcellular) = Specific intestinal permeability (transcellular)`

If you export a simulation from PK-Sim to MoBi you can see this calculation (and parameters are accessible :))
![grafik](https://user-images.githubusercontent.com/25061876/104761201-d5c80880-5762-11eb-9cf6-82119d94153d.png)


Setting calibration parameters to visible will enable users to perform the calibration directly in PK-Sim.

**Comment by StephanSchaller** [comment]
@irina-antonescu ,

yes, sorry for my misunderstanding. What I meant is that the info on the calculation for the specific int permeability is shown in the mouse-over info if you move over the entries in the dropdown (with the corresponding publication).

But I do not think the manual is "wrong" in that sense as mentioned above, but is a bit sloppy in describing the location correctly.
In principle it says you will need to come up with your own calibration and you can enter the values manually (as in the "yellow" indicated "+" in my screenshot) and later select them in the "calculation methods" window. The latter is wrongly described.

The "calculation method" window it refers to is actually the "Parameter Alternative" list in the Compound settings, when you CREATE or CONFIGURE your simulations:
![image](https://user-images.githubusercontent.com/25477643/104822678-9eca2380-5844-11eb-8b50-d282fa85e956.png)

Best,
Stephan

**Comment by irina-antonescu** [comment]
@Yuri05  thanks for your guidance, good to know how I can use MoBi & see all the equations behind the estimations!

Now I understand that what you (the manual) mean by calibration is "in vitro in vivo extrapolation" from Caco-2/PAMPA to human. So the user can do it from scratch and edit the "Intestinal permeability (transcellular)" equation in PK-Sim. However, one important thing is that for Caco-2 there are both transcellular and paracellular components included in the Papp value, so then the paracellular part should remain set to 0 (I think that is the default setting in PK-Sim) and not be refined.

What I originally thought was meant by calibration was that an Caco-2/PAMPA to human IVIVE equation is already embedded in PK-Sim, but the user can calibrate (or adjust) our own experimental Papp with the Papp of a reference compound (one that is included in the IVIVE equation in PK-Sim), to account for the interlaboratory differences between that user's lab and the lab that performed the IVIVE embedded in PK-Sim.

**Comment by irina-antonescu** [comment]
@StephanSchaller ok thanks for clarifying

**Comment by irina-antonescu** [comment]
@Yuri05 @StephanSchaller any plans for a paracellular permeation model to be added to PK-Sim any time soon? It can be an important absorption route for quite a lot of permeability-limited drugs (BCS class III and IV) and it is also biased for drugs according to ionization state.

There are some paracellular models out there where quite a lot of physiology parameters (pore size, porosity, potential drop) are refined e.g for human jejunum or Caco-2. I can provide specific examples if needed :-)

**Reply by StephanSchaller** [reply]
Hi @irina-antonescu Irina, maybe we can discuss this some time? Feel free to reach out any time!
